### DISCUSSION TITLE
Hide table row count

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1138] [Q&A] [answered]
Is there an option to hide row count displayed below a `ui.table()`

for example:
<img width="1154" alt="image" src="https://user-images.githubusercontent.com/64787868/144134054-78f7cf3e-d6a0-441a-9c26-c28aefed8d42.png">

Context: this is a"preview" of the first 10 rows of a dataset that has 39k+ rows. Though a user will hit a 'Preview' button and I can also add a message bar or some other kind of note here, the `Rows 10 of 10` can be misleading.

**Comment by mturoci** [comment, marked as answer]
Hi, hiding it is not possible. It shows the total row count in the current table.

>Though a user will hit a 'Preview' button and I can also add a message bar or some other kind of note here

Sounds good to me.

> the Rows 10 of 10 can be misleading.

I don't think so, it just tells you the count of preview data rows, which is fine.
